projects/ngx-datatable/src/lib/components/body/ghost-loader/ghost-loader.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="ghost-loader">
+    <div class="ghost-element" *ngFor="let item of [].constructor(pageSize)">
+        <div class="line" [ngStyle]="{'width': col?.width + 'px'}" *ngFor="let col of columns"></div>
+    </div>
+</div>
```

Comment:
- The library exclusively uses 2-space indents, we hence should keep it the same.
- Since you only bind a single style, we can use a direct style binding.

```suggestion
<div class="ghost-loader">
  <div class="ghost-element" *ngFor="let item of [].constructor(pageSize)">
    <div class="line" [style.width]="col?.width + 'px'" *ngFor="let col of columns"></div>
  </div>
</div>
```